typescript-id-allocator/IdCompressor.ts

```diff
@@ -12,81 +12,76 @@ import {
 	SessionSpaceCompressedId,
 	StableId,
 } from "./types";
+import { generateStableId } from "./util";
 
 export class IdCompressor implements IIdCompressor, IIdCompressorCore {
 	private readonly wasmCompressor: WasmIdCompressor;
+	public readonly localSessionId: SessionId;
 	constructor() {
-		this.wasmCompressor = new WasmIdCompressor();
+		this.localSessionId = generateStableId() as SessionId;
+		this.wasmCompressor = new WasmIdCompressor(this.localSessionId);
 	}
 
-	dispose(): void {
+	public dispose(): void {
 		this.wasmCompressor.free();
 	}
 
-	get localSessionId(): SessionId {
+	public finalizeCreationRange(range: IdCreationRange): void {
 		throw new Error("Method not implemented.");
 	}
 
-	finalizeCreationRange(range: IdCreationRange): void {
+	public takeNextCreationRange(): IdCreationRange {
 		throw new Error("Method not implemented.");
 	}
 
-	takeNextCreationRange(): IdCreationRange {
-		throw new Error("Method not implemented.");
+	public generateCompressedId(): SessionSpaceCompressedId {
+		return this.wasmCompressor.generate_next_id() as SessionSpaceCompressedId;
```

Comment:
I did implement this one-liner because one of our two TS tests uses it and was failing :D